migrator/src/main/resources/liquibase/1.0.0.Final/initial-mysql.xml

```diff
@@ -0,0 +1,424 @@
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="001" author="generated">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+            <not>
+                <tableExists tableName="AndroidVariant"/>
+            </not>
+        </preConditions>
+        <createTable tableName="AndroidVariant">
+            <column name="googleKey" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="projectNumber" type="VARCHAR(255)"/>
+            <column name="id" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet id="002" author="generated">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+            <not>
+                <tableExists tableName="ChromePackagedAppVariant"/>
+            </not>
+        </preConditions>
+        <createTable tableName="ChromePackagedAppVariant">
+            <column name="clientId" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="clientSecret" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="refreshToken" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="id" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet id="003" author="generated">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+            <not>
+                <tableExists tableName="Installation"/>
+            </not>
+        </preConditions>
+        <createTable tableName="Installation">
+            <column name="id" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="alias" type="VARCHAR(255)"/>
+            <column name="deviceToken" type="VARCHAR(4096)"/>
+            <column name="deviceType" type="VARCHAR(255)"/>
+            <column name="enabled" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
+            <column name="operatingSystem" type="VARCHAR(255)"/>
+            <column name="osVersion" type="VARCHAR(255)"/>
+            <column name="platform" type="VARCHAR(255)"/>
+            <column name="variantID" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet id="004" author="generated">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+            <not>
+                <tableExists tableName="Installation_categories"/>
+            </not>
+        </preConditions>
+        <createTable tableName="Installation_categories">
```

Comment:
And that would explain why the script ran succefully for @danielpassos and @matzew since their are under MacOS but me under Fedora where case matters.